spec/views/omniauth_callbacks/complete.html.erb_spec.rb

```diff
@@ -6,7 +6,7 @@
 describe "users/omniauth_callbacks/complete.html.erb" do
 
   let :rendered_data do
-    JSON.parse(rendered.match(/var authResult = (.*);/)[1])
+    JSON.parse(rendered.match(/data-auth-result="([^"]*)"/)[1].gsub('&quot;', '"'))
```

Comment:
hmmm so this is a bug fix?

Reply:
No, simply changed to work with rendering `authResult` in HTML data attribute.